### ISSUE TITLE
Equity/accessibility issue in Memory and Cognitive Load episode

### ISSUE THREAD

**Issue by marklcrowe** [carpentries/instructor-training#1648]
### How could the content be improved?

We discovered after the "Test your Working Memory" exercise this week than one of our participants was dyslexic and so found this activity very challenging and demotivating. 

There is an accessible option, but with the note "If you are unable to use this activity, ask your Trainer to implement the analog version of this test". This goes against Carpentries accessibility and universal design principles in that it puts the requirement on the learner to request extra support (since as a 'hidden disability' the instructors would not be aware of the need to use the analog version), and since it's a relatively quick exercise to introduce, there's not much time for them to raise issues anyway. 

Perhaps we could look out for an alternative exercise (perhaps one with spoken words already included?) to use as the default version of this.

### Which part of the content does your suggestion apply to?

https://carpentries.github.io/instructor-training/05-memory.html#test-your-working-memory

**Comment by brownsarahm** [comment]
Universal design is about having the options; there is always going to be some need for people to opt in some way.  We could reduce the disparity in opting in or shift who opts in, but having options means someone needs to select. 

That said, I think we should shift the language around this particular opt in, I think this language was written when the reason people couldn't do it was technological, when we used a flash based tool. 

Maybe we could rephrase the trainer instructions for trainers to default to running both. Maybe something like:

> for the next exercise we have two versions: an audio version and a visual version.  One 
> trainer will run the audio version in the main room, you can opt out by turning down your volume
> or using the breakout room we have opened if you prefer the visual version.  


We should also confirm that the exercise site is compatible with people using specialized fonts for dyslexia in their browser.

**Comment by ndporter** [comment]
I appreciate the feedback. If no one else jumps on this, it's is something I can probably poke around at myself and/or get feedback from accessibility colleagues, but it may be a bit before I can get to it. @brownsarahm or @marklcrowe feel free to poke me with an @ check-in if there's no movement on this by mid-July.

**Comment by ndporter** [comment]
> We should also confirm that the exercise site is compatible with people using specialized fonts for dyslexia in their browser.

I confirmed that the text displays in standard html elements, styled with a CSS body element. I'm not knowledgeable enough about web development to know what it currently takes to override CSS settings in your browser. However, the text in the box should be just as easy to modify on the user end (including its style) as any other html text.

We could actually include [OpenDyslexic](https://opendyslexic.org/about) as the stylesheet default there (or anywhere else on our sites)  as it's openly licensed and can be embedded for web use. But again, by web anything skills are ultra rusty and I'm spoiled by writing everything in MD or RMD now.

This question is probably a *put a pin in it* discussion to have with the accessibility committee at some point.

I'll try to work on the wording for the main issue sometime soon.
